src/tests/unit/stream_element_tests.ts

```diff
@@ -8,14 +8,22 @@ export class StreamElementTests extends DOMTestCase {
   }
 
   async "test action=append"() {
-    const element = createStreamElement("append", "hello", createTemplateElement(" Streams"))
+    const element = createStreamElement("append", "hello", createTemplateElement("<span> Streams</span>"))
+    const element2 = createStreamElement("append", "hello", createTemplateElement("<span> and more</span>"))
```

Comment:
Would we want to make this more clear what we are testing?

```
"<span> Element without ID</span>"
"<span> another element without ID</span>"
```

Reply:
Maybe, although since the goal of this change is to catch regressions, I'm not sure it's _that_ important in the long run. (i.e. if the code is broken at some point in the future this test will fail and it will be obvious why).
I would have updated the code, but the PR is now merged, so if you feel strongly about it feel free to open another PR for it.

Reply:
Again, opinionated feedback based on future code comprehension and readability.  If a year from now a developer is trying to understand "Why are we testing adding span elements without ID's, seems redundant.  We could simplify this test and only check if one item is added".  Adding the additional information provides clarity for future developers to understand that we are testing elements without ids.